portfolio/src/main/webapp/script.js

```diff
@@ -105,4 +106,27 @@ function changeNavbarStickiness() {
   } else {
     navbar.classList.remove("sticky");
   }
+}
+
+/*
+ * Fetches the JSON string for comments from the server and display
+ * the comments.
+ */
+function getComments() {
+  fetch("/data").then(response => response.json()).then(comments => {
+    const commentContainer = document.getElementById("comment-container");
+
+    // Build display of comments. 
+    comments.forEach((comment) => {
+      commentContainer.appendChild(createHeadingElement(comment.commenterName + "(" 
+          + comment.commenterEmail + ") says" + "\"" + comment.text +"\""));
```

Comment:
Can you put 
comment.commenterName + "(" + comment.commenterEmail + ") says" + "\"" + comment.text in a public function in Comment, and make all these 3 variables private?

Reply:
I have made the three variables private and added getter functions. From my understanding, a public function that returns comment.commenterName + "(" + comment.commenterEmail + ") says" + """ + comment.text in Comment may not be helpful since the javascript file is using the response which is in JSON format.